release-engineering/role-handbooks/branch-manager.md

```diff
@@ -430,57 +431,52 @@ This section discusses the methods in managing commits on the `release-x.y` bran
 
 During a `rc.0` release our release tooling creates a new release branch named `release-x.y`, where `x` and `y` are the major and minor versions of the next release, respectively.
 
-Behind the scenes `anago` is doing a git branch create and git push.
+Behind the scenes `krel` is doing a git branch create and git push. The branch is created in the staging phase and is pushed to the repository in the release phase.
 
 `prow`’s [`branchprotector`](https://git.k8s.io/test-infra/prow/cmd/branchprotector/README.md) runs every hour at 54 minutes past the hour and automatically adds [branch protection](https://help.github.com/articles/about-protected-branches/) to any new branch in the `kubernetes/kubernetes` repo.
 
 New release branch creation (for example: `release-1.18`) also automatically triggers an alpha.0 build for the subsequent release (for example: [`v1.19.0-alpha.0`](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.0-alpha.0)).
 
 This means that the staging step will take about twice as long, as it will stage both versions `v1.18.0-rc.0` and `v1.19.0-alpha.0`. The release step will also be extended, but not substantially longer in time.
 
-#### During staging
+#### During the release creation
 
-While the staging job is in progress, run through the following tasks, **_putting an explicit hold_** on any PRs (to be removed once the release branch has been created):
+Before the nomock release job is done, run through the following tasks, **_putting an explicit hold_** on any PRs (to be removed once the release branch has been created):
```

Comment:
Since `release` takes only 5 minutes I think it's fair to write:
```suggestion
Before the nomock release job is started, run through the following tasks, **_putting an explicit hold_** on any PRs (to be removed once the release branch has been created):
```

Reply:
Fixed.